src/_includes/partials/footer.html

```diff
@@ -92,6 +92,7 @@
             2021. Made with ❤️ by
             <a href="{{settings.url}}" target="_blank" class="underline">{{settings.author}}</a>
             •
+            <a href="https://github.com/web3community/web3community.github.io" class="underline" target="_blank" rel="noopener">View on GitHub</a>
             <a href="https://github.com/web3community/web3community.github.io" class="underline" target="_blank"
```

Comment:
Please delete this line of code , you just have to replace `h` with `H`

Reply:
Waiting